bridge/src/app.rs

```diff
@@ -59,40 +48,69 @@ impl<T: Transport> Connections<T> {
 	}
 }
 
+pub struct App<T> where T: Transport {
+	pub config: Config,
+	pub database_path: PathBuf,
+	pub connections: Connections<T>,
+	pub home_bridge: HomeBridge,
+	pub foreign_bridge: ForeignBridge,
+	pub timer: Timer,
+	pub running: Arc<AtomicBool>,
+	pub keystore: AccountProvider,
+	pub gas_prices: Arc<Mutex<GasPriceClient>>
```

Comment:
Here and in a number of places this change abandons the "trailin comma" convention. It's pretty common in Rust code and allows for cleaner diffs and often, less small mistakes.

Reply:
Ok, I can put those back in.